spec/support/fake_analytics.rb

```diff
@@ -2,10 +2,11 @@ class FakeAnalytics
   attr_reader :events
 
   def initialize
-    @events = Hash.new { |hash, key| hash[key] = [] }
```

Comment:
These changes aren't strictly related to the original ticket, but I got stuck here for some time because I was confused in seeing messages like...

```
Events received:
{"Capture Doc visited"=>[]}
```

...not realizing that the empty array was representative of "no events received by this name".